hack/app_sre_create_image_catalog.sh

```diff
@@ -23,8 +23,8 @@ git clone \
 REMOVED_VERSIONS=""
 if [[ "$REMOVE_UNDEPLOYED" == true ]]; then
     DEPLOYED_HASH=$(
-        curl -s 'https://gitlab.cee.redhat.com/service/saas-osd-operators/raw/master/splunk-forwarder-operator-services/splunk-forwarder-operator.yaml' | \
-            docker run --rm -i evns/yq -r '.services[]|select(.name="splunk-forwarder-operator").hash'
+        curl -s "https://gitlab.cee.redhat.com/service/app-interface/raw/master/data/services/osd-operators/cicd/saas/saas-splunk-forwarder-operator.yaml" | \
+            docker run --rm -i evns/yq -r '.resourceTemplates[]|select(.name="splunk-forwarder-operator").targets[]|select(.namespace["$ref"]=="/services/osd-operators/namespaces/hive-production-cluster-scope.yml")|.ref'
```

Comment:
why is this `hive-production-cluster-scope.yml` instead of `splunk-forwarder-operator-production.yml` ?

Reply:
There is no target with namespace.$ref of splunk-forwarder-operator-production.yml in https://gitlab.cee.redhat.com/service/app-interface/raw/master/data/services/osd-operators/cicd/saas/saas-splunk-forwarder-operator.yaml

I used this PR from another cluster-deployed operator's app-interface update as the template for this change: https://github.com/openshift/managed-velero-operator/pull/85/files

Reply:
ack, thx

/lgtm